inc/NRF52Pin.h

```diff
@@ -36,6 +36,7 @@ DEALINGS IN THE SOFTWARE.
 #define ID_NRF52_PIN_LO  (DEVICE_ID_IO_P0 + 36)
 
 #define NRF52PIN_PWM_CHANNEL_MAP_SIZE        4
+#define NRF52PIN_SSADC_MHANNEL_MAP_SIZE      8
```

Comment:
Typos? `NRF52PIN_SAADC_CHANNEL_MAP_SIZE`

Reply:
The nRF52 SAADC has 0 to 7 channels, which is used to specify and search channels.

Reply:
I'm just saying you have `SSADC` instead of `SAADC` and `MHANNEL` instead of `CHANNEL` in the name ;)

Reply:
oops, i will recheck that part also :) Thanks for review